File: 2210-find-target-indices-after-sorting-array/2210-find-target-indices-after-sorting-array.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var targetIndices = function(nums, target) {
    var b=[];
    var a= nums.sort((a,b)=>a-b);
    for(i=0;i<nums.length;i++){
        if(nums[i]==target){
            b.push(i)
        }
    }
    return b;
};

File: 2210-find-target-indices-after-sorting-array/2210-find-target-indices-after-sorting-array.cs
public class Solution {
    public IList<int> TargetIndices(int[] nums, int target) {
       List<int> indices=new List<int>();
       Array.Sort(nums);
       for(int i=0;i<nums.Length;i++){
        if(nums[i]==target){
            indices.Add(i);
        }
       }
       return indices;
    }
}